src/submission/submission.service.ts

```diff
@@ -119,12 +122,17 @@ export class SubmissionService {
 
     // Test Summary checks
     for (const testSumId of evaluationItem.testSumIds) {
-      const qaSuppData = await entityManager.findOneBy(QaSuppData, { testSumId });
-      if (!qaSuppData) {
+      const testSumData = await entityManager.findOneBy(TestSummary, { testSumIdentifier: testSumId });
```

Comment:
I'm not sure by the ticket if we _also_ need to check if there is a QaSuppData record?

Reply:
@mark-hayward-erg 
Mark, do we need to check both the `QaSuppData` and `TestSummary` records, or just `TestSummary`? `QaSuppData ` doesn't have the `chkSessionIdentifier` column.

Reply:
As discussed in the 5/12 standup meeting, the QaSuppData record should be checked for existence _in addition to_ confirming the existence of the check session referenced by the TestSummary record.